test/features/deploy_kogito_runtime.feature

```diff
@@ -0,0 +1,125 @@
+# Disabled until we can integrate it as part of the pipeline, see KOGITO-2278
+@disabled
+Feature: Deploy Kogito Runtime
+
+  Background:
+    Given Namespace is created
+
+  @smoke
+  Scenario Outline: Deploy Simplest Scenario using Kogito Runtime
+    Given Kogito Operator is deployed
+    And Clone Kogito examples into local directory
+    And Local example service "<example-service>" is built by Maven using profile "<profile>"
+    And Local example service "<example-service>" is deployed to image registry, image tag stored as variable "built-image"
```

Comment:
Like we do for cloning and then build, could we just store implicitely the image tag into the data context, without having to write it here ?

Reply:
That can become an issue in case we will deploy several services/images in one scenario (for example to test service discovery).

Reply:
true